PowerSet.py

```diff
@@ -0,0 +1,2 @@
+# Powerset (Set Of All Subsets) Finder.
```

Comment:
Kindly read https://github.com/girishtulabandu/One_line_PythonPrograms#contribution-guidelines before raising the pr

- Powerset already exists in the repo. Kindly delete this script from your commit also consider squashing before raisng the pr

Reply:
Hi I was having some problem due to the repo, being updated with more and more commits and not being able to squash my commits, so i'm creating a new, PR... and will delete this one as soon as you confirm.